src/app/modules/main/module.nim

```diff
@@ -874,6 +874,10 @@ method ephemeralNotificationClicked*[T](self: Module[T], id: int64) =
   else:
     self.osNotificationClicked(item.details)
 
+method onMyRequestAdded*[T](self: Module[T]) =
+    self.displayEphemeralNotification("Your Request has been submitted", "" , "checkmark-circle", false, EphemeralNotificationType.Success.int, "")
```

Comment:
General nitpick: what is the plan to make this translatable? (as this is supposedly going to end up in the UI)

Reply:
I actually don't know. I found the same usage in file. I can try to do it via qml

Reply:
How about adding the message to `CommunityRequestArgs`, will this work? Any better approach?

Reply:
Yeah, have some constant/enum value, and in QML Constants, make it translatable

Reply:
So basically, I should refactor all notifications and move all texts to qml or it looks like a `hook` :) I can create a new one issue for it, but if we planning to move from `nim` to `c++` it useless. @caybro

Reply:
> planning

Yeah, create a new issue for it; I can take a look later